x-pack/plugins/security_solution/public/timelines/store/selectors.ts

```diff
@@ -23,19 +28,22 @@ export const getTimelineShowStatusByIdSelector = () =>
     changed: timeline?.changed ?? false,
   }));
 
-const selectTimelineById = (state: State): TimelineById => state.timeline.timelineById;
+// deprecated
```

Comment:
Consider using 

```
/**
 * @deprecated REASON
 */
``` 

here instead. This will highlight the deprecation in most code editors.